examples/v1/pipelineruns/4808-regression.yaml

```diff
@@ -11,7 +11,7 @@ spec:
       type: string
   steps:
     - name: print-result
-      image: docker.io/library/bash:5.2.26
+      image: mirror.gcr.io/bash
```

Comment:
Hey @wlynch, wouldn't this still count towards docker pull quota? I could be wrong but my understanding is irrespective of where the image are hosted, they are being pulled using docker pull.